tests/components/tibber/test_services.py

```diff
@@ -246,6 +249,68 @@ async def test_get_prices_start_tomorrow(
     }
 
 
+@pytest.mark.parametrize(
+    ("start_time", "expect_it_to_work"),
+    [
+        (STARTTIME.isoformat(), True),
+        (STARTTIME.replace(tzinfo=None).isoformat(), True),
+        (
+            (STARTTIME + dt.timedelta(hours=4))
+            .replace(tzinfo=dt.timezone(dt.timedelta(hours=4)))
+            .isoformat(),
+            True,
+        ),
+        ((STARTTIME + dt.timedelta(hours=4)).isoformat(), False),
+        (
+            (STARTTIME + dt.timedelta(hours=4)).replace(tzinfo=None).isoformat(),
+            False,
+        ),
+    ],
+)
+async def test_get_prices_with_timezones(
+    freezer: FrozenDateTimeFactory,
+    start_time: str,
+    expect_it_to_work: bool,
+) -> None:
+    """Test __get_prices with timezone and without."""
+    freezer.move_to(STARTTIME)
+    call = ServiceCall(DOMAIN, PRICE_SERVICE_NAME, {"start": start_time})
+
+    result = await __get_prices(call, hass=create_mock_hass())
+
+    if expect_it_to_work:
+        assert result == {
+            "prices": {
+                "first_home": [
+                    {
+                        "start_time": STARTTIME,
+                        "price": 0.46914,
+                        "level": "VERY_EXPENSIVE",
+                    },
+                    {
+                        "start_time": STARTTIME + dt.timedelta(hours=1),
+                        "price": 0.46914,
+                        "level": "VERY_EXPENSIVE",
+                    },
+                ],
+                "second_home": [
+                    {
+                        "start_time": STARTTIME,
+                        "price": 0.46914,
+                        "level": "VERY_EXPENSIVE",
+                    },
+                    {
+                        "start_time": STARTTIME + dt.timedelta(hours=1),
+                        "price": 0.46914,
+                        "level": "VERY_EXPENSIVE",
+                    },
+                ],
+            }
+        }
+    else:
+        assert result == {"prices": {"first_home": [], "second_home": []}}
```

Comment:
Please don't use if statements in tests.
The easiest solution would be split the test into two

Reply:
I see. That's what I get for adding more test cases I guess. Fixed now.